### PR TITLE
Add: Admission controller for Checkov

### PR DESCRIPTION
**Pull request by eurogig**
Adds the admission controller features for checkov.  First commit.  

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: admissioncontroller/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (schosterbarak) [inline admissioncontroller/LICENSE:1]: no need. it can stay under the apache-2 license of the parent repo

File: admissioncontroller/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# whorf
```
Comment (schosterbarak) [inline admissioncontroller/README.md:1]: needs a small update :)

File: admissioncontroller/whorf.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
+from flask import Flask, request, jsonify
+from os import environ, remove, getenv 
+import logging
+import json
+import subprocess
+import yaml
+from datetime import datetime
+
```
Comment (schosterbarak) [inline admissioncontroller/whorf.py:8]:
```suggestion
import re

```

File: admissioncontroller/LICENSE (hunk 1 above)
Comment (eurogig) [reply admissioncontroller/LICENSE]: Great!

**Review by schosterbarak — APPROVED** [review]

**Pull request merged**